sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -21,7 +21,7 @@ using testing::Test;
 // Base class for the rest fixtures in this file.
 class AesTest : public testing::Test, public mock_mmio::MmioTest {
  public:
-  void setExpectedKey(const dif_aes_key_share_t &key, uint32_t key_size = 8) {
+  void setExpectedKey(const dif_aes_key_share_t &key, uint32_t key_size) {
```

Comment:
Not sure how this got this way, but `lowerCamelCase` is not allowed in our C++ code; this should be `UpperCamelCase()`.